data/Makefile

```diff
@@ -0,0 +1,5 @@
+motd/US/image.xbx: logo/logo_64x64.png
+	../tools/img-to-xpr.py $< $@
+
+logo/logo_64x64.png: logo/logo.svg
+	inkscape --export-png=$@ --export-width=64 --export-height=64 --export-area-drawing $<
```

Comment:
Appreciate this change a lot, however I'm not the biggest fan of introducing 3 new dependencies on the project:
- Make
- Inkscape
- Python 3

The other tools in the project are clean Rust workspaces, and I think this practice should continue ideally.

IMO, the `64x64` PNG image provided directly from the artist should be preferred over the SVG, and the xombie server should simply be able to load PNG's from disk and then swizzle it into memory at startup.

To help enable this, I've ported over your PNG script to a rust library, you can check it out here: https://github.com/DiscoStarslayer/png_to_xpr_rs

It builds as a executable, but also exposes the swizzling function as a simple library. I've verified the python script and this rust library generate identical xpr files. Feel free to include the library in this PR verbatim, or I'm open to doing the change against this branch on your local fork.

End of the day though, I feel the approach is cleaner as it allows dropping inkscape, python and make from the build requirements.

Reply:
@DiscoStarslayer 

> I'm not the biggest fan of introducing 3 new dependencies on the project

Sort of.. These tools are required only if you would like to change the artwork, but the final image is just added to the repo, so there are no added dependencies to build or launch the service.

> The other tools in the project are clean Rust workspaces, and I think this practice should continue ideally.

I agree. I was wondering how long it would take for someone to critique that this tool was not written in Rust and hopeful that person would be motivated enough to do it for me, so thanks :)

> IMO, the `64x64` PNG image provided directly from the artist should be preferred over the SVG, and the xombie server should simply be able to load PNG's from disk and then swizzle it into memory at startup.

My preference is that original art work be available in source form, and that its easy to regenerate the assets from it.

Regarding format conversion on start-up, this adds overhead of a PNG loader to SG vs just processing the logo once offline and serving it.

What I do like about your proposed approach is that it makes it easier for server admins to swap out, and down the road for images to be served dynamically, like user avatars or something.

Ultimately the convenience added to server admins probably justifies the cost of this conversion library being added to SG, so it's fine with me

> Feel free to include the library in this PR verbatim, or I'm open to doing the change against this branch on your local fork.

Frankly I'm not really motivated enough to spend more time on the feature. How about you just file a separate PR with your proposed approach